python-sdk/docs/CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 1.3.1
+
+### Feature:
+- Introduce a parameter `if_exists` which user can set to `append` or `replace` for `aql.dataframe()` operator. `append` insert into an existing table (`output_table`) from a Dataframe. [#1379](https://github.com/astronomer/astro-sdk/issues/1379)
+- Temporary Schemas accounts for Location/Region when creating BigQuery schema. User can now create schema in Bigquery as per region like - `<dataset_id>__<region_id>`. [#1369](https://github.com/astronomer/astro-sdk/issues/1369)
+
+### Bug fixes
+- Fix the `aql.cleanup()` operator as failing as the attribute `output` was implemented in 2.4.0 [#1359](https://github.com/astronomer/astro-sdk/issues/1359)
+- Fix the backward compatibility with `apache-airflow-providers-snowflake==4.0.2`. [#1351](https://github.com/astronomer/astro-sdk/issues/1351)
+- LoadFile operator returns a dataframe if not using XCom backend.[#1348](https://github.com/astronomer/astro-sdk/pull/1348),[#1337](https://github.com/astronomer/astro-sdk/issues/1337)
```

Comment:
Some of these points end with a "dot" while the others don't, I am happy with either but let's be consistent

Reply:
```suggestion
- Fix backward compatibility with `apache-airflow-providers-snowflake==4.0.2`. [#1351](https://github.com/astronomer/astro-sdk/issues/1351)
- `LoadFile` operator returns a dataframe when not using XCom backend shipped with Astro SDK [#1348](https://github.com/astronomer/astro-sdk/pull/1348),[#1337](https://github.com/astronomer/astro-sdk/issues/1337)
```